src/components/Creepypasta.stories.js

```diff
@@ -10,6 +10,8 @@ const Template = args => <Creepypasta {...args} />;
 export const Default = Template.bind({});
```

Comment:
I think this test fails because you are calling the component without MemoryRouter. Add MemoryRouter in .storybook/preview.js